css/index.css

```diff
@@ -172,4 +172,181 @@ header .logo {
 footer {
     text-align: center;
     margin: 40px 0 20px;
+}
+
+/* Media Query for Tablet devices */
+
+@media (max-width: 768px) {
+
+    .container {
+        width: 100%;
+    }
+
+
```

Comment:
This is a review from the solution lecture we had yesterday but since you did not specify the min-width on the media queries, anything you have put in 768 will cascade down to 400.